docs/src/pages/tutorial/run-in-raspberry-pi.md

```diff
@@ -44,20 +45,42 @@ npm -v
 
 ![nodejs & npm version](/tutorials/raspberry-pi/node-version-and-npm-version.png)
 
-## Run Monika in Raspbian
+## Install and Run Monika in Raspbian
 
-You can install Monika by source using git clone and npm install:
+For now, the way to install Monika in Raspbian is to install from source code.
+
+There are two ways to get Monika in Raspbian, from archive file and via Git.
+
+- Download Monika archived installer
+
+```bash
+cd ~
+wget https://github.com/hyperjumptech/monika/archive/refs/tags/v1.5.1.tar.gz
+tar xf v1.5.1.tar.gz
+cd monika-v1.5.1
+
+```
+
+- Get Monika from our Git repository
 
 ```bash
 cd ~
 git clone https://github.com/hyperjumptech/monika.git
 cd monika/
-npm install
+git checkout tags/v1.5.1 -b release-v1.5.1
```

Comment:
Better to just use shallow clone to reduce disk and bandwidth:

```
git clone --depth 1 --branch `git -c 'versionsort.suffix=-' ls-remote --exit-code --refs --sort='version:refname' --tags https://github.com/hyperjumptech/monika.git '*.*.*' | tail -1 | cut -d'/' -f 3` https://github.com/hyperjumptech/monika.git
```

The above command is to fetch the latest tag from this repo, than just clone that tag.